scram-tools.spec

```diff
@@ -0,0 +1,16 @@
+### RPM cms scram-tools 1.0
+
```

Comment:
add
```
## NOCOMPILER
```
so that it does not depend on gcc changes

Reply:
added